Tests/scripts/gitlab_slack_notifier.py

```diff
@@ -360,7 +406,7 @@ def construct_coverage_slack_msg() -> list[dict[str, Any]]:
     yesterday = datetime.now() - timedelta(days=1)
     coverage_yesterday = get_total_coverage(date=yesterday)
     color = 'good' if coverage_today >= coverage_yesterday else 'danger'
-    title = f'Content code coverage: {coverage_today:.3f}%'
+    title = f'Content code coverage: {coverage_today:.3f}% (Yesterday: {coverage_yesterday:.3f}%)'
```

Comment:
Do we have last_week and last_month?
If yes - Can you please add them here as well?